classes/components/forms/context/AccessForm.php

```diff
@@ -51,15 +51,15 @@ public function __construct($action, $locales, $context)
         }
 
         $this->addField(new FieldOptions('publishingMode', [
-            'label' => __('manager.distribution.publishingMode'),
-            'type' => 'radio',
-            'options' => [
-                ['value' => Journal::PUBLISHING_MODE_OPEN, 'label' => __('manager.distribution.publishingMode.openAccess')],
-                ['value' => Journal::PUBLISHING_MODE_SUBSCRIPTION, 'label' => __('manager.distribution.publishingMode.subscription')],
-                ['value' => Journal::PUBLISHING_MODE_NONE, 'label' => __('manager.distribution.publishingMode.none')],
-            ],
-            'value' => $context->getData('publishingMode'),
-        ]))
+                'label' => __('manager.distribution.publishingMode'),
```

Comment:
(I think these are accidental whitespace changes and can be removed.)

Reply:
Yes, added form logic initially there but it later removed but some formatting changes were left .